.github/workflows/build.yaml

```diff
@@ -12,9 +12,7 @@ on:
           [
             'main',
             'master',
-            {name: 'beta', prerelease: true},
```

Comment:
We should have a look if we can add support for alpha/beta using release-it. It does have some configuration for it, but have not tested it yet. Is there a repo where we are actively using this now?